Homework/src/by/tms/Lesson_7/cards/ForeignAlfaCard.java

```diff
@@ -0,0 +1,19 @@
+package by.tms.Lesson_7.cards;
+
+public class ForeignAlfaCard extends AlfabankCard {
+
+    private String currency;
+
+    public ForeignAlfaCard(String name, String currency) {
```

Comment:
ForeignAlfaCard, ForeignBPSCard  and ForeignBelarusCard again - similar. You should left only one of them, or update them to be diferent